### PR TITLE
Properly enable systemd services

### PR DESCRIPTION
**Pull request by ramereth**
On systems which have proper ganeti systemd services, properly enable them so
that the daemons run properly on reboot.

In addition, this includes some other fixes:

- Set the epel repo to our mirror so we don't run into errors during testing
- Enable drbd on master suite
- Remove Berksfile.lock so cookbooks get updated each time for testing
- Set node['ganeti']['master-node'] to false since nil does not equate to false
- Use openssh cookbook in test cookbook so that we don't run into verify issues
- Use hostname if fqdn doesn't exist in test cookbook
- Update ServerSpec to deal with cron test on debian platforms

### CODE REVIEW

**Comment by ramereth** [comment]
[Properly enable Ganeti services on newer versions](https://trello.com/c/M6eMjzfE/632-properly-enable-ganeti-services-on-newer-versions)

File: attributes/default.rb [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+  ganeti-confd
+  ganeti-kvmd
+  ganeti-luxid
+  ganeti-noded
+  ganeti-rapi
+  ganeti-wconfd
+)
 default['ganeti']['drbd'] = true
-default['ganeti']['master-node'] = nil
+default['ganeti']['master-node'] = false
```
Comment (TravisWhitehead) [inline attributes/default.rb]: You might want to update the README, as it still says `nil` is the default value.

**Comment by ramereth** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.5.0.  Have a nice day!

**Pull request merged**